pkg/api/volumes.go

```diff
@@ -147,12 +147,16 @@ func (client *Client) ShowVolumes(volume string) ([]common.VolumeObject, *common
 		}
 	} else {
 		logger.V(4).Info("-- ShowVolumesNamesGet", "status", httpRes.Status, "err", err, "body", httpRes.Body)
+		return nil, &common.ResponseStatus{}, err
 	}
 
 	logger.V(4).Info("================================================================================")
 
 	returnVolumes := []common.VolumeObject{}
-	status := CreateCommonStatus(logger, &response.Status)
+	status := &common.ResponseStatus{}
```

Comment:
If response.Status is nil we return a default common.ResponseStatus(). Please look into either a) setting some values for common.ResponseStatus(), or b) changing CreateCommonStatus to handle a nil response.Status and filling in the correct common.ResponseStatus() to be returned to the caller.